tests/integration/model/test_models.py

```diff
@@ -176,11 +182,23 @@ def _get_objects(
             tuple[AnomalyModule, AnomalibDataModule, Engine]: Returns the created objects for model, dataset,
                 and engine
         """
+        # Mock the ChatGPTWrapper if the model_name is "chat_g_p_t_vision"
+        if model_name == "chat_g_p_t_vision":
+            os.environ["OPENAI_API_KEY"] = "fake-api-key"
+            with (
+                patch("anomalib.models.image.chatgpt_vision.chatgpt.ChatGPTWrapper") as mock_chatgptwrapper,
+            ):
+                mock_instance = mock_chatgptwrapper.return_value
+                mock_instance.api_call.return_value = "NO"
+                self.mock_chatgptwrapper = mock_chatgptwrapper  # Store the mock for potential later use
```

Comment:
@samet-akcay Would you also like to refactor the `patch(...) as mock...`
to not use `from unittest.mock import patch`